mp/src/public/view_shared.h

```diff
@@ -112,6 +120,14 @@ class CViewSetup
 	// (0.0f means use the viewport)
 	float		m_flAspectRatio;
 
+	// Camera settings to control motion blur
```

Comment:
You are changing layout of this structure, it is directly passed from/to engine

Reply:
You're right, didn't realise this was used by engine. Thankfully xen figured out that these members are only used by the SFM motion blur setting which we don't need, so I can safely remove them without affecting the motion blur.